src/app/src/components/VectorTileFacilitiesLayer.jsx

```diff
@@ -108,30 +238,81 @@ const VectorTileFacilitiesLayer = ({
         throw new Error('Missing tile cache key');
     }
 
+    const handleVectorLayerClick = (data) => {
+        try {
+            setMultipleFacilitiesAtPoint(null);
+
+            const facilitiesAtSamePoint = findFacilitiesAtSamePointFromVectorTile(
+                data,
+                vectorTileLayerRef,
+            );
+
+            if (facilitiesAtSamePoint.length < 1) {
+                return noop();
+            }
+
+            if (facilitiesAtSamePoint.length === 1) {
+                return handleMarkerClick(data);
+            }
+
+            setMultipleFacilitiesAtPoint(facilitiesAtSamePoint);
+            return setMultipleFacilitiesAtPointPosition(data.latlng);
+        } catch (e) {
+            window.console.error(e);
```

Comment:
Is this left over from debugging? Does this propagate to rollbar or should we not catch the exception so that it does get logged to rollbar?

Reply:
I believe this is not propagated to Rollbar. I added this whole try/catch block because I don't want a mistake in the click code to crash the entire map, but I left the console.error line so that we could easily debug it if it occurred on production.

Would it be worthwhile to log it to Rollbar?